### DISCUSSION TITLE
Combining PF scatter plots for phases with different crystal symmetries

### DISCUSSION THREAD

**Discussion by dalemw** [mtex-toolbox/mtex#794] [Ask Anything]
Hi all,

Is it possible to overlay pole figure (PF) scatter plots for phases with different crystal symmetries? Using the code below I encounter errors due to the symmetries not matching.

`figure()
plotPDF(ebsd('wurtzite').orientations,Miller(0,0,1,ebsd('wurtzite').CS))
hold on
plotPDF(ebsd('zincblende').orientations,Miller(1,1,1,ebsd('zincblende').CS))
hold off`

I can achieve the intended effect using photo editing programmes (example attached), but it would be ideal if there was a solution within MATLAB/MTEX.

![pb_1110001](https://user-images.githubusercontent.com/79329745/108544524-82575600-72de-11eb-8224-b559136e35ca.png)

**Comment by kilir** [comment]
Hi,
just don't use  `plotPDF` and compute the directions for each pole figure:
```
v1 = ebsd('wurtzite').orientations*Miller(0,0,1,ebsd('wurtzite').CS) % probably no need to symmetrise here
v2 = ebsd('zincblende').orientations*symmetrise(Miller(1,1,1,ebsd('zincblende').CS),'unique','antipodal') % here you need to take all symmetrically equivalent directions

plot(v1)
hold on
plot(v2)
hold off
```

Cheers,
Rüdiger
